deps/build.jl

```diff
@@ -0,0 +1,15 @@
+import Conda
+#Workaround the MacOS build issue when building the package with Conda 
+
+ if !haskey(ENV, "PYTHON") || ENV["PYTHON"] == "" || ENV["PYTHON"] == "Conda"
+    if Sys.isapple()  
+      #use non-mkl versions of python packages
+        Conda.add("nomkl")
+        Conda.rm("mkl")
+        #installs scikit-learn if not already installed
+        PyCall.pyimport_conda("sklearn", "scikit-learn") 
+    else
+      #installs scikit-learn if not already installed
+        PyCall.pyimport_conda("sklearn", "scikit-learn")
+    end
```

Comment:
The philosophy so far is that you can use `ScikitLearn.jl` without installing `scikit-learn`. Couldn't you put this `Conda.add("nomkl")` logic in `sk_import` instead of `build.jl`?

Reply:
I think i get your logic. Is this because in some cases the user may want to use pure `julia` models in `ScikitLearn.jl` ??.


> Couldn't you put this `Conda.add("nomkl")` logic in `sk_import` instead of `build.jl`?

I think i might be able to make that possible with a `try` statement

Reply:
> . Is this because in some cases the user may want to use pure julia models in ScikitLearn.jl 

Yes, exactly. I've done it before. If you're willing to write your own `fit/transform` for your own model, you don't need `scikit-learn` at all.